staging/src/k8s.io/apiserver/pkg/util/webhook/webhook_test.go

```diff
@@ -163,15 +163,15 @@ func TestKubeConfigFile(t *testing.T) {
 			errRegex: fmt.Sprintf(errMissingCertPath, "certificate-authority", badCAPath, "", badCAPath),
 		},
 		{
-			test: "cluster with invalid CA certificate ",
+			test: "cluster with invalid CA certificate",
 			cluster: &v1.NamedCluster{
 				Cluster: v1.Cluster{
 					Server:                   namedCluster.Cluster.Server,
-					CertificateAuthorityData: caKey,
+					CertificateAuthorityData: caKey, // pretend user put caKey here instead of caCert
 				},
 			},
 			user:     &defaultUser,
-			errRegex: "", // Not an error at parse time, only when using the webhook
+			errRegex: "unable to load root certificates",
```

Comment:
This is actually the case I ran into downstream (user put in invalid CA data and did not realize it).